### PR TITLE
Add Ubuntu 22.04 Automatus workflow

### PR DESCRIPTION
**Pull request by alanmcanonical**
#### Description:

- Add Ubuntu 22.04 Automatus workflow

#### Rationale:

- Add this workflow so that we can build and test content on Ubuntu 22.04

### CODE REVIEW

**Comment by dodys** [comment]
@marcusburghardt @Mab879 would it be ok to add the automatus workflow for ubuntu? And could you verify it is in order like the rest?

File: .github/workflows/automatus-ubuntu2204.yaml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          prop_path: 'ansible'
+      - uses: actions/download-artifact@v4
+        if: ${{ steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        with:
+          name: ${{ env.DATASTREAM }}
+      - name: Run tests in a container - Bash
+        if: ${{steps.bash.outputs.prop == 'True' && steps.ctf.outputs.CTF_OUTPUT_SIZE != '0' }}
+        run: tests/test_rule_in_container.sh --no-remove-machine-only --dontclean --logdir logs_bash --remediate-using bash --name ssg_test_suite --datastream $DATASTREAM ${{join(fromJSON(steps.rules.outputs.prop))}}
+        env:
+          ADDITIONAL_TEST_OPTIONS: "--duplicate-templates --remove-fips-certified --product rhel8"
```
Comment (Mab879) [inline .github/workflows/automatus-ubuntu2204.yaml:139]:
```suggestion
          ADDITIONAL_TEST_OPTIONS: "--duplicate-templates --remove-fips-certified"
```
This is a workaround for Fedora and RHEL

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR.

I found a few things that need some attention.

**Review by Mab879 — APPROVED** [review]
LGTM.

Thanks.

**Pull request merged**